underfs/pom.xml

```diff
@@ -16,6 +16,7 @@
     <module>glusterfs</module>
     <module>swift</module>
     <module>s3</module>
+    <module>oss</module>
```

Comment:
NIT: those modules are sorted alphabetically. Could you put `oss` before `swift`?